phoenix-core/src/main/java/org/apache/phoenix/exception/SQLExceptionInfo.java

```diff
@@ -47,6 +51,10 @@
     private final String familyName;
     private final String columnName;
     private final String functionName;
+    private final long maxMutationSize;
+    private final int mutationSize;
```

Comment:
This should be "long" if variable maxMutationSize is also long or vice-a-versa

Reply:
Same for Builder class too

Reply:
nice catch here @mihir6692

Reply:
Thanks @mihir6692 @yanxinyi I had kept it "long" but mutationSize(numRows) in MutationState class is an integer ([here](https://github.com/apache/phoenix/blob/356f4cd7d43bccb9538a5a2b94863b1c52cd9aad/phoenix-core/src/main/java/org/apache/phoenix/execute/MutationState.java#L139)), hence I changed it to int.

Reply:
I think this is the relevant [code path ](https://github.com/apache/phoenix/blob/356f4cd7d43bccb9538a5a2b94863b1c52cd9aad/phoenix-core/src/main/java/org/apache/phoenix/execute/MutationState.java#L130). 

But based on this [QueryServicesOptions](https://github.com/apache/phoenix/blob/78cfb911a157fa74fd9868099e20df3eae6f62ba/phoenix-core/src/main/java/org/apache/phoenix/query/QueryServicesOptions.java#L147) Number of max rows in single batch are limited to int range.

What do you think? @yanxinyi @ChinmaySKulkarni

Reply:
since we have a `DEFAULT_MAX_MUTATION_SIZE = 500000`, it's ok to use `int` instead of `long` from my point of view. What do you think @ChinmaySKulkarni

Reply:
@nsgupta1 I think it's fine to keep both to `int` too. But, then for consistency, also better to make the data type change in other relevant places too as @mihir6692 mentioned.

Reply:
Changes maxSize variable to int from long for consistency.